PKGBUILD

```diff
@@ -46,11 +47,19 @@ ros_depends=(ros-noetic-tf2-ros
   ros-noetic-tf2
   ros-noetic-eigen-conversions)
 depends=(${ros_depends[@]}
+  geographiclib
   eigen)
 
 _dir="robot_localization-${pkgver}"
-source=("${pkgname}-${pkgver}.tar.gz"::"https://github.com/cra-ros-pkg/robot_localization/archive/${pkgver}.tar.gz")
-sha256sums=('9ca03e611b3c0d024dca3642a8eb0da75a8a776c6dee3082717a314e41fbafa8')
+source=("${pkgname}-${pkgver}.tar.gz"::"https://github.com/cra-ros-pkg/robot_localization/archive/${pkgver}.tar.gz"
+        "624.patch"::"https://github.com/cra-ros-pkg/robot_localization/pull/624.patch")
+sha256sums=('67d566a1d2af92699f1a6d93c3b1672f2fc305ce651a47825cc2584c3fe5a78f'
+            'f3414f3571590d0f58f8772126850e6d747405265e286a86a7a14ec956b306be')
```

Comment:
Oh also I think I missed this in the last one (its fine), but for pull requests I think we like to have `SKIP` in the `sha256sums` entry so that if the upstream PR changes we don't have to manually change the checksum.

Reply:
Done